### DISCUSSION TITLE
P1001: Can't reach database server at `db.[dbname].supabase.co`:`5432`

### DISCUSSION THREAD

**Discussion by Marcus100** [supabase/supabase#21031] [Questions] [answered]
Hello, I am using the Remix Epic Stack and this tutorial to integrate Prisma and Supabase (https://supabase.com/partners/integrations/prisma). 
When I run the command [] I get the error

"npx prisma migrate dev --name init
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "db.[dbname].supabase.co:5432"

Error: P1001: Can't reach database server at `db.[dbname].supabase.co`:`5432`

Please make sure your database server is running at `db.[dbname].supabase.co`:`5432`."

I switched the actual db name. Unsure if I should share it publicly. The dev server works fine when I run "npm run dev", and the data from my Supabase tables load.

There are two other posts on this, but it doesn't provide a solution to my problem.
https://github.com/orgs/supabase/discussions?discussions_q=P1001%3A+Can%27t+reach+database+server

**Comment by GaryAustin1** [comment, marked as answer]
You likely need to be using Supavisor now.  Everything transitioned to ipv6 last week.  
Please see the latest connections here:  https://supabase.com/partners/integrations/prisma
And for lots of info this discussion: https://github.com/orgs/supabase/discussions/17817

**Comment by Marcus100** [comment]
Thank you! I reread the document properly and solved it. I was confusing the DIRECT_URL and DATABASE_URL.
